### DISCUSSION TITLE
Camera not auto-starting

### DISCUSSION THREAD

**Discussion by sneakocom** [mebjas/html5-qrcode#967] [Q&A]
Hello,

I have the QR code app setup and running (works great, thank you!) but need it to start scanning as soon as the page loads. Right now it requires me to press "Start Scanning" to bring up the camera.  Is it possible to start the camera immediately?

Thank you.

**Comment by ROBERT-MCDOWELL** [comment]
no, not the first time, browsers (each reacts differently) need user confirmation, most of the time, to avoid any spy app or such
